cardano_node_tests/tests/test_cli.py

```diff
@@ -26,6 +26,11 @@
 
 ADDR_ALPHABET = list(f"{string.ascii_lowercase}{string.digits}")
 
+SKIPIF_STAKE_ADDRESS_OPTION_UNUSABLE = pytest.mark.skipif(
```

Comment:
It is better to implement this as a fixture, so the `cardano-cli` command is executed only when the test is about to be run, and not during tests collection. We want to do as little work during tests collection as possible. Tests collection can happen for various reasons, like building documentation.
See e.g. https://github.com/input-output-hk/cardano-node-tests/blob/be11c7a7f9e92fe612125e7c13408fc7021f931a/cardano_node_tests/tests/test_blocks.py#L32-L35